tests/Makefile.am

```diff
@@ -29,15 +29,15 @@ EXTRA_DIST=test_utils.h \
 	lattices/example_cvp_out2 \
 	lattices/example_cvp_out3 \
 	lattices/example_cvp_out4 \
-	lattices/example_cvp_out5 
+	lattices/example_cvp_out5
```

Comment:
Does your editor delete trailing whitespaces unconditionally?

Reply:
I'm not sure if it's if the editor or my previous screw-up with clang-format. :s